wiki/Beatmap_search/de.md

```diff
@@ -0,0 +1,118 @@
+| `played` | Die Zeit in Tagen, seit wann zuletzt gespielt wurde. |
+| `unplayed` | Zeigt nur ungespielte Beatmaps an. Dabei muss ein Vergleich mit einem leeren Wert verwendet werden (z. B. `unplayed=`). Der Vergleich selbst wird ignoriert. |
+| `speed` | Gespeicherte Scrollgeschwindigkeit in osu!mania. Immer 0 für ungespielte Beatmaps oder wenn die Option [`osu!mania-Scrollgeschwindigkeit für jede Beatmap merken`](/wiki/Client/Options#gameplay) ausgeschaltet ist. |
+
+## Website[^website-filters]
+
+| Filter | Beschreibung |
+| :-: | :-- |
+| `artist` | Name des Künstlers |
+| `featured_artist` | Kennung eines [Featured Artist](/wiki/People/Featured_Artists) |
+| `creator` | Name des Beatmap-Erstellers |
+| `ar` | [Approach-Rate](/wiki/Beatmap/Approach_rate) |
+| `cs` | [Circle-Size](/wiki/Beatmap/Circle_size) |
+| `od` | [Allgemeine Schwierigkeit](/wiki/Beatmap/Overall_difficulty) |
+| `hp`, `dr` | [HP Drain-Rate](/wiki/Beatmap/HP_drain_rate) |
+| `star`, `stars` | [Sternebewertung](/wiki/Beatmap/Star_rating) |
+| `bpm` | [Songtempo](/wiki/Music_theory/Tempo) |
+| `length` | [Länge](/wiki/Beatmap/Play_time) in Sekunden |
+| `key`, `keys` | Anzahl der Keys (betrifft nur osu!mania und konvertierte Beatmaps) |
+| `status` | Beatmap-Status. Der Wert kann `ranked`, `approved`, `pending`, `notsubmitted`, `unknown` oder `loved` sein bzw. `r`/`a`/`p`/`n`/`u`/`l` in Kurzform. |
+| `created` | Erstellungsdatum der Beatmap |
+| `ranked` | Datum, an dem die Beatmap gerankt/approved wurde |
+
+## Client (lazer)[^lazer-filters]
+
+| Filter | Beschreibung |
+| :-: | :-- |
+| `artist` | Name des Künstlers |
+| `title` | Songtitel |
+| `creator` | Name des Beatmap-Erstellers |
+| `ar` | [Approach-Rate](/wiki/Beatmap/Approach_rate) |
+| `cs` | [Circle-Size](/wiki/Beatmap/Circle_size) |
+| `od` | [Allgemeine Schwierigkeit](/wiki/Beatmap/Overall_difficulty) |
+| `hp`, `dr` | [HP Drain-Rate](/wiki/Beatmap/HP_drain_rate) |
+| `star`, `stars` | [Sternebewertung](/wiki/Beatmap/Star_rating) |
+| `bpm` | [Songtempo](/wiki/Music_theory/Tempo) |
+| `length` | [Länge](/wiki/Beatmap/Play_time) in Sekunden |
+| `key`, `keys` | Anzahl der Keys (betrifft nur osu!mania und konvertierte Beatmaps) |
+| `status` | Beatmap-Status. Der Wert kann `ranked`, `approved`, `pending`, `notsubmitted`, `unknown` oder `loved` sein bzw. `r`/`a`/`p`/`n`/`u`/`l` in Kurzform. |
+
+## Beispielhafte Anfragen
+
+Finde Beatmaps mit geringer OD, die in einen bestimmten Schwierigkeitsbereich fallen:
+
+```
+stars>=7 stars<8 od<8
+```
+
+(Client) Zeige gerankte Beatmaps zum Thema Weihnachten an, die bisher nicht gespielt wurden:
+
+```
+unplayed= status=r christmas
+```
+
+(Client, Webseite) Prüfe, ob mindestens ein Level von ::{ flag=FR }:: [galvenize](https://osu.ppy.sh/users/381444) aus [DJ Fresh - Gold Dust](https://osu.ppy.sh/beatmapsets/28107) heruntergeladen wurde:
```

Comment:
In a Discord server I'm in, many people expressed that the current word that is used for `difficulty` (Schwierigkeitsgrad) is never used by anyone and sounds too unwieldly. I was thinking that using `Level` more frequently might be a good alternative.

Before, I already have sometimes used this translation, though for transparency I'm noting it here.

Reply:
I'm not really talking to a lot of people about osu! in German so I don't know what terms are actually more used than others, but level seems like a fine translation either way :+1: